crypto/rand/rand_vms.c

```diff
@@ -32,10 +32,21 @@
 #  pragma message disable DOLLARID
 # endif
 
+# include <dlfcn.h>              /* SYS$GET_ENTROPY presence */
+
 # ifndef OPENSSL_RAND_SEED_OS
 #  error "Unsupported seeding method configured; must be os"
 # endif
 
+/*
+ * DATA COLLECTION METHOD
+ * ======================
+ *
+ * This is a method to get *some* very low quality entropy.
+ * It works by collecting all kinds of statistical data that
+ * VMS offers and using them as "entropy".
```

Comment:
Placing terms in double quotes gives them a dubious emphasis (at that's the case in german language.)

```
... and using them as random seed.
```

Reply:
Sure